### PR TITLE
Update next.config.js

### LINKED ISSUE #487: Create aliases for the imports to omit long import statements (dev issue not end user's)
**Problem:**
In the Screen Shot below we have a very long import statement which disrupts the code appearance and also make importing components harder which can be done relatively easy manner.

code

Can be seen in these files as well : 

Screenshot from 2024-07-07 23-36-36

**Solution**
We can use path aliasing like `@components/AppbarClient` there are some alias already present inside _next.config.js_ and we can describe some more and use it as a standard to bring uniformity in the code base.

correct imports examples : 

code

### PR DESCRIPTION
**Pull request by S007K**
### PR Fixes:
- 1 very long import statement which disrupts the code appearance and also make importing components harder which can be done relatively easy manner by adding  resolve aliases in next config.
- 2 fixes #487 

Resolves #487 

### Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

File: apps/web/next.config.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
 /** @type {import('next').NextConfig} */
+const path = require('path');
 const webpack = require('webpack');
 
 module.exports = {
   webpack: (config) => {
+
+      // Add path aliases
+    config.resolve.alias = {
```
Comment (devsargam) [inline apps/web/next.config.js:9]: does this not break existing codebase? If not going to mege it

Comment (S007K) [reply apps/web/next.config.js:9]: You can use aliases in new files or new parts of your codebase while keeping relative paths in existing files. This allows you to gradually transition to using aliases.

Comment (S007K) [reply apps/web/next.config.js:9]: You can have a mix of imports in your codebase. For example, some files might use import Button from '@components/Button' while others might use import Button from '../../components/Button'. This approach is perfectly valid.

Comment (S007K) [reply apps/web/next.config.js:9]: you can think of aliases in configuration as a form of syntactic sugar. They provide a more convenient and readable way to handle imports, especially in larger projects.

**Comment by devsargam** [comment]
testing locally

**Comment by devsargam** [comment]
Waiting for harkirat's confirmation on this one. As I think this repo can be demonorepofied this would be useless.

**Pull request merged**